docs/design/scheduler-configuration.md

```diff
@@ -99,13 +99,13 @@ Adding user based limits will allow the cluster administrators to control the cl
     * Maximum (resource)
 
 ### Placement Rules definition
-Dynamically placing an application in a queue has been a feature of the schedulers on YARN for a long time. This means that an application when submitted does not have to include a queue to run in. 
+Dynamically placing an application in a queue has been a feature of schedulers for a long time. This means that an application when submitted does not have to include a queue to run in. 
 
 A placement rule will use the application details to place the application in the queue. The outcome of running a placement rule will be a fully qualified queue or a `fail`, which means execute the next rule in the list. Rules will be executed in the order that they are defined.
 
 During the evaluation of the rule the result could be a queue name that contains a dot. This is especially true for user and group names which are POSIX compliant. When a rule generates a partial queue name that contains a dot it must be replaced as it is the separator in the hierarchy. The replacement text will be `_dot_`
 
-The first rule that matches, i.e. returns a fully qualified queue name, will halt the execution of the rules. If the application is not placed at the end of the list of rules the application will be rejected. Rules can return queues that are not defined in the configuration only if the rule allows creation of queues. These queues created by the placement rules are considered _unmanaged_ queues.
+The first rule that matches, i.e. returns a fully qualified queue name, will halt the execution of the rules. If the application is not placed at the end of the list of rules the application will be rejected. Rules can return queues that are not defined in the configuration only if the rule allows creation of queues. These queues created by the placement rules are considered _unmanaged_ queues as they are not managed by the administrator in the configuration.
```

Comment:
does this deserve a separate paragraph or some highlight to explain `unmanaged` vs `managed`? when I read the doc, I am a bit confused by the terms and always can't tell from one from another.